src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java

```diff
@@ -99,4 +101,16 @@
         return new ResponseEntity <>(new MessageDTO(lockedException.getMessage()), HttpStatus.METHOD_NOT_ALLOWED);
     }
 
+    @ExceptionHandler(ConstructorException.class)
+    ResponseEntity <MessageDTO> handleConstructorException(ConstructorException constructorException) {
```

Comment:
I predict a question durring the demo: what is that?
Please try to remove this method.

I believe you should catch this exception in the place it is expected to be thrown and rethow a custom exception or just IllegalArgumentException. The last one is already handled here.

Reply:
You can use EmptyFileException with different messages.